documentation/New-PnPSiteCollectionTermStore.md

```diff
@@ -0,0 +1,67 @@
+---
+applicable: SharePoint Online
+external help file: PnP.PowerShell.dll-Help.xml
+Module Name: PnP.PowerShell
+online version: https://pnp.github.io/powershell/cmdlets/new-pnpsitecollectiontermstore
+schema: 2.0.0
+title: New-PnPSiteCollectionTermStore
+---
+
+# Add-PnPSiteCollectionTermStore
```

Comment:
almost ;-)

Reply:
Damn, you've sure got a crisp eye :) Fixed!